chapter_09_commands.asciidoc

```diff
@@ -404,7 +401,7 @@ Let's look again at the 'handle_event' method from our message bus.
 [[messagebus_logging]]
 .Current handle function (src/allocation/service_layer/messagebus.py)
 ====
-[source,python,highlight=4;7]
+[source,python,highlight=8;12]
```

Comment:
Not sure what lines are supposed to be highlighted, but these will get the logger lines.